src/studio-home/tabs-section/libraries-v2-tab/WelcomeLibrariesV2Alert.tsx

```diff
@@ -0,0 +1,47 @@
+import { Alert, Button } from '@openedx/paragon';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+
+import { useLibrariesV1Data } from '@src/studio-home/data/apiHooks';
+
+import messages from '../messages';
+
+const libraryDocsLink = (
+  <Alert.Link href="https://docs.openedx.org/en/latest/educators/how-tos/course_development/create_new_library.html">
+    <FormattedMessage {...messages.alertLibrariesDocLinkText} />
+  </Alert.Link>
+);
+
+export const MigrateLegacyLibrariesAlert = () => {
+  const { data, isLoading, isError } = useLibrariesV1Data();
+
+  // Does not show the alert if we are still loading or if there was an error fetching libraries
+  if (isLoading || isError) {
+    return null;
+  }
+
+  const hasPendingV1Migrations = data.libraries.some(library => !library.isMigrated);
+  return (
+    <Alert variant="info">
+      {hasPendingV1Migrations && (
+        <Alert.Heading>
+          <FormattedMessage {...messages.alertTitle} />
+        </Alert.Heading>
+      )}
+      {hasPendingV1Migrations ? (
```

Comment:
Is there a reason for having the heading inside a separate condition instead of moving it below inside of ternary operator?

Reply:
No! My bad!
At first I thought I would reuse some of the body, but it was not the case. Fixed here: b28059c9ad319a8bc9773620ef5a84027fede448